.github/workflows/tests.yml

```diff
@@ -1,29 +1,39 @@
 name: Tests
 
 on:
-  push:
-    branches:
-      - main
-      - raye
-  pull_request:
-  schedule:
-    # Every Friday at 1:00am UTC
-    - cron: '0 1 * * 5'
-  workflow_dispatch:
-
+  workflow_call:
+    inputs:
+      disable-vcs:
+        required: true
+        type: string
+      repository:
+        required: true
+        type: string
+        
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+
+
```

Comment:
Max one empty line in between section (applies to all yml files)

Reply:
Added empty line.